publishable/database/migrations/2016_01_01_000000_add_user_avatar_and_role_id.php

```diff
@@ -6,25 +6,23 @@ class AddUserAvatar extends Migration
 {
     /**
```

Comment:
Renaming this file makes a issue with the classname, the classname must match a part of the filename.